### PR TITLE
Feature/toggle boxes

### PR DESCRIPTION
**Pull request by virtuoso98**
Features:
1. Added support to toggle outline for bounding box + opacity (Annotator Settings, Cogwheel) with some CSS

Bugfixes:
1. Solved issue where cogwheel at bottom right (annotator settings) do not appear
2. Attempted to resolve issue where annotator playback is still happening, I referred to https://wicg.github.io/video-rvfc/
3. Solved CSS issue where video-thumbnail has a huge width and. Please test by loading a few videos and images and seeing whether the thumbnail behaves normally

### CODE REVIEW

File: src/app/src/components/annotations/annotator.tsx [hunk 1]
```diff
@@ -607,7 +632,7 @@ export default class Annotator extends Component<
       }
 
       this.setState({ predictDone: 0, uiState: null });
-    }, 750);
+    }, 0);
```
Comment (beesaycheese) [inline src/app/src/components/annotations/annotator.tsx]: What's the reason for this change? If it is going to be 0, I don't think we will need the setTimeout anymore

File: src/app/src/components/annotations/annotator.tsx [hunk 2]
```diff
@@ -841,13 +873,20 @@ export default class Annotator extends Component<
...
         this.annotationGroup.removeLayer(layer);
       });
       this.updateMenuBarAnnotations();
+      if (videoElement) {
+        (videoElement as any).cancelVideoFrameCallback(
+          this.state.currAnnotationPlaybackId
+        );
+      }
     }
+    console.log(this.state.currAnnotationPlaybackId);
```
Comment (beesaycheese) [inline src/app/src/components/annotations/annotator.tsx]: don't need this console

**Review by beesaycheese — CHANGES_REQUESTED** [review]

File: src/app/src/components/annotations/annotator.tsx (hunk 1 above)
Comment (virtuoso98) [reply src/app/src/components/annotations/annotator.tsx]: This change was done so that it could "simulate" the call of an API. Since this is not needed anymore, I have removed it.

File: src/app/src/components/annotations/annotator.tsx (hunk 2 above)
Comment (virtuoso98) [reply src/app/src/components/annotations/annotator.tsx]: Removed console.log

File: src/app/src/components/annotations/annotator.tsx (hunk 1 above)
Comment (beesaycheese) [reply src/app/src/components/annotations/annotator.tsx]: Ok got it!

**Review by beesaycheese — APPROVED** [review]
Thanks for making the change. Approving this PR

File: src/app/src/components/annotations/annotator.tsx [hunk 3]
```diff
@@ -270,6 +273,8 @@ export default class Annotator extends Component<
     this.filterAnnotationVisibility = this.filterAnnotationVisibility.bind(
       this
     );
+    this.setAnnotationOutline = this.setAnnotationOutline.bind(this);
```
Comment (gkeechin) [inline src/app/src/components/annotations/annotator.tsx]: @virtuoso98 considering that the state is a single object that describes the annotation aesthetics, can you combine them into `this.setAnnotationOptions` and process this option by taking in both variables?

File: src/app/src/components/annotations/utils/annotatorsettings.tsx [hunk 4]
```diff
@@ -41,11 +48,29 @@ export default class AnnotatorSettings extends Component<
...
+    ) {
+      return document.querySelector(
+        ".leaflet-pane.leaflet-overlay-pane img.leaflet-image-layer"
+      );
+    }
+    return document.querySelector(
+      ".leaflet-pane.leaflet-overlay-pane video.leaflet-image-layer"
+    );
+  };
+
```
Comment (gkeechin) [inline src/app/src/components/annotations/utils/annotatorsettings.tsx]: @virtuoso98 I understand the intuition, but you might be more clear doing ternary if you are sure that the element on the screen is binary.

```javascript
private getImageElement = (): any => document.querySelector(
        ".leaflet-pane.leaflet-overlay-pane img.leaflet-image-layer"
      ) ?? document.querySelector(
      ".leaflet-pane.leaflet-overlay-pane video.leaflet-image-layer"
    );

Comment (gkeechin) [reply src/app/src/components/annotations/utils/annotatorsettings.tsx]: Using JavaScript Nullish Coalescing Operator - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

**Review by gkeechin — CHANGES_REQUESTED** [review]

File: src/app/src/components/annotations/utils/annotatorsettings.tsx (hunk 4 above)
Comment (virtuoso98) [reply src/app/src/components/annotations/utils/annotatorsettings.tsx]: Changes done. Thanks for the shorthand!

File: src/app/src/components/annotations/annotator.tsx (hunk 3 above)
Comment (virtuoso98) [reply src/app/src/components/annotations/annotator.tsx]: I have done this based on your advice, adding in a switch case: 
image

**Review by gkeechin — APPROVED** [review]
